torch/quantization/quantize.py

```diff
@@ -19,7 +24,9 @@ def propagate_qconfig_helper(module, qconfig_dict, qconfig_parent=None, prefix='
     Return:
         None, module is modified inplace with qconfig attached
     """
-    if not hasattr(module, 'qconfig'):
+    if type(module) in skip_list:
+        assert not hasattr(module, 'qconfig'), str(type(module)) + ' module is in skip list, it should not have qconfig'
```

Comment:
This is not a good user experience. Can we ignore qconfig for modules in skip list?

Reply:
you mean silently skip these modules?